uvicorn/middleware/wsgi.py

```diff
@@ -36,8 +49,8 @@ def build_environ(scope, message, body):
         environ["REMOTE_ADDR"] = client[0]
 
     # Go through headers and make them into environ entries
-    for name, value in scope.get("headers", []):
-        name = name.decode("latin1")
+    for name_, value_ in scope.get("headers", []):
```

Comment:
Can we use `key, value` instead? And on the `value` below, `value_str`? Or just maintain the same `name, value` here, but using the `name_str` and `value_str` below?